app/config/init.js

```diff
@@ -14,6 +15,7 @@ const _syntaxValidation = function (cfg) {
   try {
     return new vm.Script(cfg, {filename: '.hyper.js', displayErrors: true});
   } catch (err) {
+    notify(`Configuration Error! Please fix ~/.hyper.js!`);
```

Comment:
There's no need for this since one line below we show a more in-depth notification.